mail/postfix/Makefile

```diff
@@ -116,6 +136,16 @@ ifdef CONFIG_POSTFIX_SQLITE
   AUXLIBS+=-L$(STAGING_DIR)/usr/lib -lsqlite3 -lpthread
 endif
 
+ifdef CONFIG_POSTFIX_PGSQL
+  CCARGS+=-DHAS_PGSQL -I$(STAGING_DIR)/usr/include/
+  AUXLIBS+=-L$(STAGING_DIR)/usr/lib -lpq
+endif
+
+ifdef CONFIG_POSTFIX_PCRE
+  CCARGS+=-DHAS_PCRE -I$(STAGING_DIR)/usr/include/
```

Comment:
Do we need `-DHAS_PCRE` in the affirmative case?  The build directions:

http://www.postfix.org/INSTALL.html#build_other

seems to imply that we only need to call it out in the negative case: `-DNO_PCRE`.